core/jazz_ui/src/app/pages/environment-detail/environment-detail.component.html

```diff
@@ -22,6 +22,24 @@ <h1 class="page-hdr bold-title relative">{{friendly_name}}
 
                 <span class="page-sub-title">{{service.name}}</span><br>
             </div>
+            <div class="image-btn-div">
```

Comment:
Noting that these should be image assets. I will commit to your branch an update shortly.